.github/workflows/crease_release.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - master
+    - 'releases/**'
```

Comment:
is there a reason this file is named crease_release?

Reply:
Lol. A typo. Let me push a fix, once I have tested it.